samples/cellular/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -367,6 +368,171 @@ To monitor the GNSS module (for instance, to see whether A-GPS or P-GPS assistan
+
+First, download and install the `nRF Cloud Utilities <nRF Cloud Utilities documentation_>`_ repo.
+
+Next, use the nRF Cloud Utilities ``create_ca_cert.py`` script to generate a self-managed CA certificate:
+
+.. code-block:: console
+
+   python3 create_ca_cert.py -c US -st StateOrProvince -l MyCity -o MyCompany -ou RD -cn "mywebsite.com" -f self_ca_
+
+The ``-f self_ca_`` argument will cause the files output by ``create_ca_cert.py``to be prefixed with ``self_ca_``.
+Rember to populate the ``-c``, ``-st``, ``-l``, ``-o``, ``-ou``, and ``-cn`` arguments with your own values.
+See `Create CA Cert <nRF Cloud Utilities Create CA Cert_>`_ for more details.
+
+If this succeeds, you should now have three files:
+
+  * ``self_ca_XXXXX_ca.pem``
+  * ``self_ca_XXXXX_prv.pem``
+  * ``self_ca_XXXXX_pub.pem``
+
+where ``XXXXX`` is a hex code unique to your certificate.
+You will use these files to sign device certificates.
+
+You only need to generate these three files once.
+They can be used to sign as many device certificates as you need.
+
+Next, decide on a device ID for each device you wish to provision.
+The ID can be anything you want, as long as it is not prefixed with ``nrf-``.
+For example, we will use ``my_wifi_dev`` as the device ID for the duration of these instructions.
+You must consistently use this exact device ID in the remaining steps, otherwise provisioning will fail.
+
+Next, generate and self-sign a device certificate for each device you wish to provision:
+
+.. code-block:: console
+
+   python3 create_device_credentials.py -ca self_ca_XXXXX_ca.pem -ca_key self_ca_XXXXX_prv.pem -c US -st StateOrProvince -l MyCity -o MyCompany -ou RD -cn my_wifi_dev -dv 2000 -f cred_
+
+Where ``self_ca_XXXXX_ca.pem``, ``self_ca_XXXXX_prv.pem`` are replaced by the corresponding outputs from the previous step.
+Make sure that the ``-cn`` argument is set to your chosen device ID.
+The ``-f cred_`` argument ensures all outputs from ``create_device_credentials.py`` are prefixed with ``cred_``.
+The ``-dv 2000`` specifies that the certificate should remain valid for 2000 days.
+Rember to populate the ``-c``, ``-st``, ``-l``, ``-o``, and ``-ou`` arguments with your own values.
+
+If this succeeds, you should now have an additional three files:
+
+  * ``cred_my_wifi_dev_crt.pem``
+  * ``cred_my_wifi_dev_pub.pem``
+  * ``cred_my_wifi_dev_prv.pem``
+
+These three files can only be used by a single device.
+The filenames automatically include your chosen device ID.
+If an attacker obtains the private key (``cred_my_wifi_dev_prv.pem``), they will be able to impersonate your device.
+Since the :kconfig:option:`CONFIG_NRF_CLOUD_PROVISION_CERTIFICATES` does not protect this private key from extraction, it should not be used in production!
+
+Next, create a file named ``my_wifi_dev_cred.csv`` and give it the following contents:
+
+.. code-block:: csv
```

Comment:
CSV is not a valid option for code-block, and it is breaking the doc build. You can just leave it empty.